### PR TITLE
Sort collections in tree-view

### PR DESCRIPTION
**Pull request by mmarcon**
## Description
Sorts collections in alphanumerical order.

### Checklist
- [ ] New tests and/or benchmarks are included
- [ ] Documentation is changed or added

## Motivation and Context

I expect collections to be in alphanumerical order, otherwise I have to scan them all to find the one I need.

- [ ] Bugfix
- [ ] New feature
- [ ] Dependency update
- [x] Misc

## Open Questions
- Not sure why they don't come back already in the right order.

## Types of changes

- [ ] *Backport Needed*
- [x] Patch (non-breaking change which fixes an issue)
- [ ] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by alenakhineika** [comment]
Maybe it would be nice to have a test for it?

**Comment by mmarcon** [comment]
@alenakhineika you are right. Let me add a test in a little bit.

**Comment by mmarcon** [comment]
In fact, looks like I made a test fail. I will do the testing part to relax a bit once I am done with drafting the demo script.

**Comment by Anemy** [comment]
Hmmm those might not be from this work - I got the same failures on my pr.
Ah yup - I'll fix in https://github.com/mongodb-js/vscode/pull/43

File: src/templates/playgroundTemplate.ts [hunk 1]
```diff
@@ -1,27 +1,15 @@
...
 
 // Run a find command.
-db.sales.find({ date: { $gte: new Date("2014-04-04"), $lt: new Date("2014-04-05") } });
+db.myCollection.find({ foo: 'bar' });
 
 // Run an aggregation.
-const aggregation = [
-  { $match: { date: { $gte: new Date("2014-01-01"), $lt: new Date("2015-01-01") } } },
-  { $group: { _id : "$item", totalSaleAmount: { $sum: { $multiply: [ "$price", "$quantity" ] } } } }
+const agg = [
```
Comment (Anemy) [inline src/templates/playgroundTemplate.ts]: Maybe aggPipeline? agg is good too.

Comment (mmarcon) [reply src/templates/playgroundTemplate.ts]: This is just a revert of a commit that should not have been in this PR

**Pull request merged**